cv32/sim/uvmt_cv32/README.md

```diff
@@ -93,10 +93,10 @@ Running the envrionment with Metrics [dsim](https://metrics.ca)
 ----------------------
 The command **make SIMULATOR=dsim sanity** will run the sanity testcase using _dsim_.
 <br><br>
-Setting a shell environment variable `SIMULATOR` to "dsim" will also define the
+Setting a shell environment variable `CV_SIMULATOR` to "dsim" will also define the
 Makefile variable SIMULATOR to `dsim` and you can save yourself a lot of typing. For
 example, in a bash shell:
-<br>**export SIMULATOR=dsim**
+<br>**export CV_SIMULATOR=dsim**
```

Comment:
Groan.  I know why you are doing this, but its gonna cause heartburn.   Nevertheless, let's merge it in now and get it over with.   We should prep a slide to explain these changes for the Tuesday status meeting.